ssas/service/server.go

```diff
@@ -187,32 +187,23 @@ var ttlScale = time.Minute
 // CommonClaims contains the superset of claims that may be found in the token
 type CommonClaims struct {
 	jwt.StandardClaims
+	// AccessToken, MFAToken, or RegistrationToken
+	TokenType string	 `json:"typ,omitempty"`
```

Comment:
the 'typ' field is specified to be a header field indicating the media type (e.g., jwt, jwks) of the token. We should not use it to name a different field. Maybe 'use'?

https://tools.ietf.org/html/rfc7519#section-5